core/signer/signer.go

```diff
@@ -115,20 +114,59 @@ func (s *Signer) ProcessMutation(ctx context.Context, txn transaction.Txn, index
 		return fmt.Errorf("QueueLeaf err: %v", err)
 	}
 
-	mutationObj := new(tpb.SignedKV)
-	if err := proto.Unmarshal(mutation, mutationObj); err != nil {
-		return err
+	log.Printf("Sequenced %x", index)
+	return nil
+}
+
+// processMutations reads mutations from the database and adds them to the tree.
+// processMutations returns the maximum mutation sequence number processed.
+func (s *Signer) processMutations(ctx context.Context, txn transaction.Txn) (uint64, error) {
+	startSequence := uint64(0)
+	smh := new(ctmap.SignedMapHead)
+	if _, _, err := s.sths.Latest(ctx, smh); err == nil {
+		startSequence = smh.GetMapHead().MaxSequenceNumber + 1
```

Comment:
Just thinking about what this would mean for timestamps... 
- Timestamps use the time uncertainty at a particular moment to figure out what +1 means. 
- I think there are APIs for making open / closed interval requests, eg. (  ], [ ], [ ), ( )

Consider defining the `ReadAll` API to be a ( ] interval?